sdk/eventhub/azure-eventhub-checkpointstoretable/azure/eventhub/extensions/checkpointstoretable/_tablestoragecs.py

```diff
@@ -165,16 +178,18 @@ def _update_ownership(self, ownership, **kwargs):
                 entity=ownership_entity, headers={"Prefer": "return-content"}, **kwargs
             )
             ownership["etag"] = metadata["etag"]
+            print('swathi')
+            print(metadata["content"]["Timestamp"])
```

Comment:
did you forget to apply the change or push the changes? FYI: I see `print` is still here 🤣

Reply:
yep, forgot to push the changes, sorry about that